moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  
+  <!-- startup simulated world -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" default="worlds/empty.world"/>
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="gui" value="$(arg gui)"/>
+  </include>
+
+  <!-- send robot urdf to param server -->
+  <param unless="$(arg limited)" name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+
+  <param if="$(arg limited)" name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+
+  <!-- push robot_description to factory and spawn robot in gazebo -->
+  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model robot -z 0.1" respawn="false" output="screen" />
```

Comment:
please break this line into two (line too long - max is 120)

Reply:
Thanks for the review, will fix the changes.

Btw I got to get franka to be spawned in gazebo, but I had to do some changes to the urdf file first. What would you recommend, do we make the changes that makes a urdf model gazebo combitable a part of this PR? Or do we better make it a separate one and make this one just for the launch file?

Reply:
please move this discussion to the main issue or the gdoc, it does not belong inline with a PR code review :-)